Makefile

```diff
@@ -119,7 +119,7 @@ build.version:
 	@echo "$(VERSION)" > $(OUTPUT_DIR)/version
 
 build.common: export SKIP_GEN_CRD_DOCS=true
-build.common: build.version helm.build mod.check crds gen-rbac
+build.common: build.version helm.build mod.check crds gen-rbac fetch-csi-operator-files
```

Comment:
As discussed, let's not make this part of the build. The csi operator files are out of the control of the rook repo. They can change at any time independently from Rook. This would cause breakage in the master and release branches. Instead, the decision to update the csi files needs to be an explicit decision by an engineer. A similar pattern is how we have updated the csi manifests in the past, usually each time there is a stable csi release, and with a goal to integrate a specific version. So may not need any `make` command to run the script, we can just run the script directly.

Reply:
Yes, I have removed make file changes and it will be just a script